boot/zephyr/CMakeLists.txt

```diff
@@ -161,6 +155,12 @@ if(CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256 OR CONFIG_BOOT_ENCRYPT_EC256)
       ${MBEDTLS_ASN1_DIR}/src/asn1parse.c
       ${MBEDTLS_ASN1_DIR}/src/platform_util.c
       )
+  # Since here we are not using Zephyr's mbedTLS but rather our own, we need
+  # to set MBEDTLS_CONFIG_FILE ourselves. When using Zephyr's copy, this
+  # variable is set by its Kconfig in the Zephyr codebase.
+  zephyr_library_compile_definitions(
```

Comment:
Even though CONFIG_BOOT_USE_MBEDTLS is not enabled still it uses mcuboot-mbedtls-cfg.h, is it fine??

Reply:
Yes .As if MBEDTLS is not enabled then the code flow comes here if tinycrpt is enabled so mcuboot can use the same file also key based approach (ecdsa and rsa ) uses the same file